client/allocrunner/csi_hook.go

```diff
@@ -549,7 +558,8 @@ func (c *csiHook) Shutdown() {
 // or when a -dev mode client is stopped. Cancel our shutdown context
 // so that we don't block client shutdown while in the CSI RPC retry
 // loop.
-func (c *csiHook) Destroy() {
+func (c *csiHook) Destroy() error {
 	c.logger.Trace("destroying hook")
```

Comment:
Unrelated but we already have trace level logs for every hook in alloc_runner_hooks.go
```suggestion
```

Reply:
Oh, good catch.. Done here and the `Shutdown` hook.